src/main/java/uk/gov/hmcts/reform/civil/HighLevelDataSetupApp.java

```diff
@@ -97,16 +97,8 @@ public void createRoleAssignments() {
     }
 
     @Override
-    protected boolean shouldTolerateDataSetupFailure(Throwable e) {
-        int httpStatusCode504 = 504;
-        if (e instanceof ImportException) {
-            ImportException importException = (ImportException) e;
-            return importException.getHttpStatusCode() == httpStatusCode504;
-        }
-        if(e instanceof SSLException){
-            return true;
-        }
-        return false;
+    protected boolean shouldTolerateDataSetupFailure() {
+     return true;
```

Comment:
Is it correct to skip all errors, this code is used for production high level data setup as well